relayer/channel.go

```diff
@@ -112,8 +104,8 @@ func ExecuteChannelStep(src, dst *Chain) (success, last, modified bool, err erro
 	}
 
 	// Query Channel data from src and dst
-	srcChan, dstChan, err = QueryChannelPair(src, dst, int64(sh.GetHeight(src.ChainID))-1,
-		int64(sh.GetHeight(dst.ChainID))-1)
+	srcChan, dstChan, err = QueryChannelPair(src, dst, int64(srcUpdateHeader.GetHeight().GetRevisionHeight())-1,
```

Comment:
ah I will update this to use the helper functions since it reads better (I made the helper functions after fixing this)